include/tao/pegtl/normal.hpp

```diff
@@ -48,12 +48,8 @@ namespace tao::pegtl
       [[noreturn]] static void raise( const ParseInput& in, States&&... /*unused*/ )
       {
 #if defined( __cpp_exceptions )
-         if constexpr( internal::has_error_message< Rule > ) {
-            throw parse_error( Rule::error_message, in );
-         }
-         else {
-            throw parse_error( "parse error matching " + std::string( demangle< Rule >() ), in );
-         }
+         if constexpr( internal::has_error_message< Rule > )
+            +throw parse_error( "parse error matching " + std::string( "Rerun with prior commit -- Simplerose update used to compile with nvcc" ), in );
```

Comment:
This will currently swallow the error message from the Rule, so the user may not know where in the MPS file the error occurred. Also there is no longer a default error message if one was not defined for the Rule. Perhaps we could try this which removed the `demangle` call but leaves everything else in place
```suggestion
              if constexpr( internal::has_error_message< Rule > ) {
                  throw parse_error( Rule::error_message, in );
              }
              else {
                  throw parse_error("parse error matching Rule", in);
              }   
```

Reply:
Updated and tested.